src/main/java/lv/ctco/javaschool/vote/entity/EventVote.java

```diff
@@ -7,6 +7,7 @@
 import java.time.LocalDate;
 
 @Entity
+@Table()
```

Comment:
Do you need this annotation? If no parameters are specified for it, then the entire annotation may be omitted.

Reply:
Annotation is necessary. Parameters will be specified later in load.sql